client/app/application.coffee

```diff
@@ -87,9 +78,21 @@ class App extends Application
 
 
     # Handler for register route, display onboarding's current step
-    handleRegisterRoute: =>
+    initializeOnboarding: =>
         # Load onboarding stylesheet
         AppStyles = require './styles/onboarding.styl'
+        # onboarding steps config
+        steps = require './config/steps/all'
+
+        user = {
+            username: ENV.username,
+            hasValidInfos: ENV.hasValidInfos
+        }
+
+        @onboarding = new Onboarding(user, steps, ENV.currentStep)
```

Comment:
Again, here this code will be called every time the route change, for every step. I think we should to the initialization only once. I propose a code like this one:

```coffeescript````
initializeOnboarding: ->
    steps = require './config/steps/all'

    user = {
        username: ENV.username,
        hasValidInfos: ENV.hasValidInfos
    }

    onboarding = new Onboarding(user, steps, ENV.currentStep)
    onboarding.onStepChanged (step) => @handleStepChanged(step)
    onboarding.onStepFailed (step, err) => @handleStepFailed(step, err)
    onboarding.onDone () => @handleTriggerDone()

    return onboarding


# Handler for register route, display onboarding's current step
handleRegisterRoute: =>
    @onboarding ?= @initializeOnboarding()

    # Load onboarding stylesheet
    AppStyles = require './styles/onboarding.styl'

    currentStep = @onboarding.getCurrentStep()
    @router.navigate currentStep.route
    @showStep(currentStep)

``